src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java

```diff
@@ -111,14 +112,24 @@ public static String getPagesPackage() {
      * @return the timeOut
      */
     public static int getTimeOut() {
+        if (TIMEOUT == null) {
+            if (Props.get("page.load.timeout") == null || "".equals(Props.get("page.load.timeout"))) {
+                LOG.warn("Set timeout in your properties file, key 'page.load.timeout'. Now using default value "
+                        + defaultTimeout + " milliseconds.");
```

Comment:
Can we use build-in formatting instead of strings contat?
Like this:
LOG.warn("Set timeout in your properties file, key 'page.load.timeout'. Now using default value {} milliseconds.", defaultTimeout );